app/src/main/java/com/example/polina/socialnetwork/PostDeteilsActivity.java

```diff
@@ -0,0 +1,331 @@
+package com.example.polina.socialnetwork;
+
+import android.content.Intent;
+import android.net.Uri;
+import android.support.v7.app.ActionBarActivity;
+import android.os.Bundle;
+import android.view.ContextMenu;
+import android.view.LayoutInflater;
+import android.view.MenuInflater;
+import android.view.MenuItem;
+import android.view.View;
+import android.view.ViewGroup;
+import android.webkit.CookieManager;
+import android.webkit.CookieSyncManager;
+import android.widget.AdapterView;
+import android.widget.CheckBox;
+import android.widget.CompoundButton;
+import android.widget.EditText;
+import android.widget.ImageView;
+import android.widget.ListView;
+import android.widget.TextView;
+
+import com.android.volley.AuthFailureError;
+import com.android.volley.Request;
+import com.android.volley.Response;
+import com.android.volley.VolleyError;
+import com.android.volley.toolbox.ImageLoader;
+import com.android.volley.toolbox.NetworkImageView;
+import com.android.volley.toolbox.StringRequest;
+import com.android.volley.toolbox.Volley;
+
+import org.androidannotations.annotations.AfterViews;
+import org.androidannotations.annotations.App;
+import org.androidannotations.annotations.Background;
+import org.androidannotations.annotations.EActivity;
+import org.androidannotations.annotations.UiThread;
+import org.androidannotations.annotations.ViewById;
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+
+@EActivity(R.layout.activity_post_deteils)
+public class PostDeteilsActivity extends ActionBarActivity {
+
+    @App
+    SNApp snApp;
+    @ViewById(R.id.comments)
+    ListView commentsList;
+    @ViewById(R.id.edit_comment)
+    EditText newComment;
+
+
+    NetworkImageView imageUser;
+    TextView userName;
+    TextView postDate;
+    TextView postText;
+    NetworkImageView imagePost;
+    CheckBox checkBoxLike;
+    TextView likeCount;
+    ImageView location;
+    TextView commentsCount;
+
+    Intent intent;
+    private String postID;
+    String idUser;
+    private ImageLoader mImageLoader;
+    ArrayList<User> usersLiked;
+    private CommentsAdapter adapter;
+    ViewGroup header;
+    ArrayList<Comment> comments = new ArrayList<Comment>();
+    int countComments;
+    int countLikes;
+    int INTENT_EDIT = 0;
+
+
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        intent = getIntent();
+        postID = intent.getStringExtra(Utils.IDPOST);
+        System.out.println(postID);
+        mImageLoader = Utils.getmImageLoader(this);
```

Comment:
make imageLoader that will be shared between PostActivity and PostDetailActivity